deploy/clusterrole.yaml

```diff
@@ -59,6 +59,7 @@ rules:
   - configmaps
   - secrets
   - namespaces
+  - poddisruptionbudgets
```

Comment:
this goes in `policy` not here.  This is already covered as well in the existing clusterrole.yaml:

```
- apiGroups:
  - policy
  resources:
  - poddisruptionbudgets
  verbs:
  - '*'
```

Reply:
Please remove this from the or thx

Reply:
Please remove this from the or thx

Reply:
ok I didn't see it is added in policy group already. But somehow I am still seeing permission issue for pdb when testing.

Reply:
can you paste the error?  I would like to see if in some way i fubared this section.. :)